custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,146 @@
+            self._state.get("dhcp_leases", [])
+        )
+
+        # calcule pps and kbps
+        update_time = dict_get(self._state, "update_time")
+        previous_update_time = dict_get(self._state, "previous_state.update_time")
+
+        if previous_update_time is not None:
+            elapsed_time = update_time - previous_update_time
+
+            for interface_name, interface in dict_get(
+                self._state, "telemetry.interfaces", {}
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)